Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1026,6 +1026,13 @@ static void loadPrimitive(
       !options.pMeshOptions->pNodeOptions->pModelOptions
            ->ignoreKhrMaterialsUnlit;
 
+  // We can't calculate flat normals for points or lines, so we have to force
+  // them to be unlit if no normals are specified. Otherwise this causes a
+  // crash when attempting to calculate flat normals.
+  if (primitive.mode < MeshPrimitive::Mode::TRIANGLES && !hasNormals) {
```

Comment:
@kring @csciguy8 Purely style question, is it okay to hardcode checks like this where the values of spec constants are assumed? Or is it better to be more explicit, e.g.:

```
bool isTriangles = primitive.mode == MeshPrimitive::Mode::TRIANGLES ||
   primitive.mode == MeshPrimitive::Mode::TRIANGLE_FAN || 
   primitive.mode == MeshPrimitive::Mode::TRIANGLE_STRIP;
   
if (!isTriangles && !hasNormals) {
```

Reply:
If I had to chose, I think I would prefer explicit.

With a < operator and enums, you're implicitly assuming a conversion to an integer, and in addition a specific order in the enum. Explicit avoids this.

Also, if this enum ever switched to the more modern scoped enum (enum class X), which doesn't as easily allow casts to an integer, explicit becomes a more pragmatic choice..

Reply:
Given that these constants come from a formal spec, they're extremely unlikely to change. So the original code should be pretty safe, and a tiny bit more performant. But it's a little clearer if it's explicit. You could get the best of both worlds (at the cost of more typing) by using a `static_assert` to prove the values are what you expect. And this would also serve as a bit of documentation. Probably overkill in this case, though. Probably just bike shedding here, I'm basically ok with any of the approaches.